bedrock/security/templates/security/partials/cve.html

```diff
@@ -1,19 +1,21 @@
-<h4 id="{{ id }}" class="level-heading cve">
-  <a href="#{{ id }}"><span class="anchor">#</span>{{ id }}: {{ title }}</a>
-</h4>
-<dl class="summary">
-  <dt>Reporter</dt>
-  <dd>{{ reporter|safe }}</dd>
-  <dt>Impact</dt>
-  <dd><span class="level {{ impact_class }}">{{ impact }}</span></dd>
-</dl>
-<h5>Description</h5>
-{{ description|markdown }}
-{% if bugs %}
-  <h5>References</h5>
-  <ul>
-    {% for bug in bugs -%}
-      <li><a href="{{ bug.url }}">{{ bug.desc|safe }}</a></li>
-    {%- endfor %}
-  </ul>
-{% endif %}
+<div class="cve">
+  <h4 id="{{ id }}" class="level-heading">
+    <a href="#{{ id }}"><span class="anchor">#</span>{{ id }}: {{ title }}</a>
+  </h4>
+  <dl class="summary">
+    <dt>Reporter</dt>
+    <dd>{{ reporter|safe }}</dd>
+    <dt>Impact</dt>
+    <dd><span class="level {{ impact_class }}">{{ impact }}</span></dd>
+  </dl>
+  <h5>Description</h5>
+  {{ description|markdown }}
+  {% if bugs %}
+    <h5>References</h5>
```

Comment:
It's not actually a sub heading of "Description", it's just indented because of the `if` tag. In that case `h5` is correct right?